source/jormungandr/jormungandr/parking_space_availability/bss/cykleo.py

```diff
@@ -137,17 +137,25 @@ def __repr__(self):
         return ('cykleo-{}'.format(self.network)).encode('utf-8', 'backslashreplace')
 
     def get_informations(self, poi):
+        # Possible status values of the station: IN_SERVICE, IN_MAINTENANCE, OUT_OF_SERVICE, DISCONNECTED
+        # and DECOMMISSIONED
         ref = poi.get('properties', {}).get('ref')
         if ref is not None:
             ref = ref.lstrip('0')
         data = self._call_webservice()
         if not data:
-            return None
+            return Stands(0, 0, 'Unavailable')
         station = data.get(ref)
         if not station:
-            return None
-        if 'availableDockCount' in station \
-                and ('availableClassicBikeCount' in station or 'availableElectricBikeCount' in station):
+            return Stands(0, 0, 'Unavailable')
+
+        if station.get('station', {}).get('status') == 'IN_SERVICE' and 'availableDockCount' in station:
```

Comment:
- How is the response formatted? station.get('status') doesn't work?

- Why check only 'availableDockCount' ?

Reply:
if lattribut 'availableDockCount' is the only absent in the response, we create a station with status='Unavailable' and other attributes initialized to zero.

Reply:
if lattribut 'availableDockCount' is the only absent in the response, we create a station with status='Unavailable' and other attributes initialized to zero.